.github/workflows/release.yml

```
@@ -22,13 +22,24 @@ jobs:
           node-version: 20
           registry-url: https://registry.npmjs.org
 
+      - name: Install Bun
+        if: ${{ steps.release.outputs.release_published == 'true' }}
```

actionlint (1.7.7)

property "release" is not defined in object type {}

(expression)